databuilder/utils/math_utils.py

```diff
@@ -0,0 +1,8 @@
+import operator
+
+
+def floordiv(num_lhs, num_rhs):
+    """
+    NOTE: Match behaviour by SQL that rounds towards zero for negative numbers.
+    """
+    return operator.floordiv(num_lhs, num_rhs)
```

Comment:
I tried and I think I  succeeded in defining a function that matches the SQL behaviour, but it was long with a couple if else statements ... I used `int()` but any pointers would be great